### ISSUE TITLE
Compiled Executable Errors. 

### ISSUE THREAD

**Issue by noisycat05** [noisycat05/Backroom-net#11]
# Issue:

Windows 10 x64

No error logs are ever produced with this error(s).
The database file is a likely cause along with File paths.

So, just open up the application and press the one key, (Currently used as create package.)
If you try to compress anything, It will not find the file and will not compress or do anything really.
If you try to exit regularly, it crashes, if you try to press exit, it crashes and if you try to open it again, it crashes. it only runs properly in IDE.

**Comment by noisycat05** [comment]
Right at the moment, I can ask Purebasic to create an executable of the actual IDE test program each time I run it with the IDE and it works fine. So it will create a temporary executable and That works fine when running separately. I just don't think I should do that and release it to anyone because the executable is open to manipulation.

**Comment by noisycat05** [comment]
Issues where narrowed down to Cryptographic module.
Also getting an,
```Purebasic
(Warning) The stack is no more 16 bytes aligned !
```
Error form the BackRoom keyboard function.
I started to use the Purebasic Purifier.

**Comment by noisycat05** [comment]
Issue fixed in Cryptographic module.
```Purebasic
Repeat
  If Keyboard(conplace) <> 0
    Debug conplace
    If msg$ = "esc"
      MessageRequester("BackRoom-Beta-1.0.0","User Hit Escape Key. Please Wait for shutdown.")
      CleanShutDown()
    EndIf
       Select msg$
        Case "1"
          SpredDir("null",EncryptStorage() \MasterMem, *keyMem)
          ClearConsole()
          Goto men
        EndSelect
  EndIf
  
  
Until Exit = 1
```
Now when we call console after the fact when we create a package, it gives the 16 bytes warning.

**Comment by noisycat05** [comment]
```Purebasic
[10:22:51] [ERROR] Invalid memory access. (read error at address 18446744073709551615)
```

**Comment by noisycat05** [comment]
Issue was fixed in 60fca3e08c3ac9a3ac6d225cd1842ea2e4170447 - thanks @crodzilla !

**Comment by noisycat05** [comment]
# Errors Solved
Issue will be closed.
